spec/custom_matchers.rb

```diff
@@ -0,0 +1,12 @@
+RSpec::Matchers.define :belong_to do |expected|
+  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+end
```

Comment:
Ah, sorry, my bad. But aren't tests too weak, if we only check "ok, _some_ subset of data should be there"?
Like, you know, empty dataframe also will pass the check ;)